### DISCUSSION TITLE
PostgreSQL VS MySql ?

### DISCUSSION THREAD

**Discussion by alex001x** [opendcim/openDCIM#1516] [General]
Hello Opendcim team,

Question:  PostgreSQL compatible with Opendcim (replaces mysql) ?
In a VM installation with CentOS + postgreSQL, is there any contraindication for installing Opendcim ?

Thks, Alexandre 
FR

**Comment by wilpig** [comment]
Not supported.  We started a project to normalize the SQL to make it not mysql/mariadb specific but we've never finished that process.  It will likely work but there is no telling what function won't.
